build/checksums/nodejs.txt

```diff
@@ -0,0 +1,34 @@
+dfb37570ef34ac04f34c26d0ec558df60a9665df5961c01c1657c0ca495f2f01  node-v16.17.1-aix-ppc64.tar.gz
```

Comment:
Maybe there should be a file comment with instructions and/or gulp task to update this when we change node.js versions?

Reply:
The idea is to drop in this file from the download mirror.